src/compiler/transformers/decorators-to-static/convert-decorators.ts

```diff
@@ -116,15 +114,14 @@ const removeStencilMethodDecorators = (
   diagnostics: d.Diagnostic[]
 ): ts.ClassElement[] => {
   return classMembers.map((member) => {
-    const currentDecorators = member.decorators;
-    const newDecorators = filterDecorators(member.decorators, MEMBER_DECORATORS_TO_REMOVE);
+    const currentDecorators = retrieveTsDecorators(member);
+    const newDecorators = filterDecorators(currentDecorators, MEMBER_DECORATORS_TO_REMOVE);
```

Comment:
We have `newDecorators ?? []` twice below and no other references to it (other than a `!==` check), should we maybe DRY it up a little by doing something like this?

```suggestion
    const newDecorators = filterDecorators(currentDecorators, MEMBER_DECORATORS_TO_REMOVE) ?? [];
```

then I think we could just use `newDecorators` directly below, having ensured it's going to be an array.

Reply:
ahh true, carry on 😆